ci-operator/step-registry/ipi/conf/aws/proxy/ipi-conf-aws-proxy-commands.sh

```diff
@@ -0,0 +1,356 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
```

Comment:
You'll want the usual:

```sh
trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
```

in here to handle "my command was TERMed and should now reap it's children before exiting".